openstack/sap-seeds/templates/_flavors_hana_no_exclusive.tpl

```diff
@@ -93,20 +86,17 @@
   disk: 64
   is_public: true
   extra_specs:
-    "vmware:hv_enabled": "True"
+    {{- tuple . "vmware" | include "sap_seeds.helpers.extra_specs" | indent 4 }}
     "hw_video:ram_max_mb": "16"
```

Comment:
`hw_video:ram_max_mb` is included in the tuple and can be removed

Reply:
Removed it